core/plugins/org.fusesource.ide.camel.model.service.core/src/org/fusesource/ide/camel/model/service/core/util/versionmapper/CamelForFuseOnOpenShiftToBomMapper.java

```diff
@@ -49,6 +49,7 @@ public CamelForFuseOnOpenShiftToBomMapper() {
 		mapping.put(FUSE_750_CAMEL_VERSION, "3.0.11.fuse-750035-redhat-00001");
 		mapping.put(FUSE_760_CAMEL_VERSION, "3.0.11.fuse-760025-redhat-00001");
 		mapping.put(FUSE_770_CAMEL_VERSION, "3.0.11.fuse-770009-redhat-00002");
+		mapping.put(FUSE_780_CAMEL_VERSION, "3.0.11.fuse-770009-redhat-00002");
```

Comment:
seems the value has not been updated, it is the 7.7

Reply:
yes, I did not find a newer one at https://maven.repository.redhat.com/ga/io/fabric8/fabric8-project-bom-camel-spring-boot/

Reply:
I think it is that it is now using the community version 3.0.12 based on https://github.com/jbosstools/jbosstools-fuse/pull/1639#discussion_r545903170 so I think that 3.0.12 must be set here.

Reply:
@apupier @lhein while "Fuse 7 for Spring Boot 1" BOM indeed imported `io.fabric8/fabric8-project-bom-camel-spring-boot` BOM, "Fuse 7 for Spring Boot 2" BOM switched to import of `org.jboss.redhat-fuse/fabric8-project-bom-camel-spring-boot`.
The difference is that for SB1, the helper BOM was generated using sundr-maven-plugin, while in SB2, Andrea Tarrochi and Tom Cunningam moved this to manually crafted helper BOM. See https://github.com/jboss-fuse/redhat-fuse/commit/f8b61588.

I can sense the purpose of `CamelForFuseOnOpenShiftToBomMapper.java `, but IMO, this _intermediary_ BOM should not be used and instead - official (full) "Fuse 7 Spring Boot 2 BOM" should be used. Which means `org.jboss.redhat-fuse/fuse-springboot-bom` group/artifact IDs.

Reply:
the new `org.jboss.redhat-fuse/fuse-springboot-bom` is the one used in the provided templates in the Eclipse desktop based tooling.
External users can provide their own example. They might still use the old groupId/artifactId. As they are still provided at runtime level, I think we need to continue to support it also.

Reply:
but as @lhein pointed out, there's no more `io.fabric8/fabric8-project-bom-camel-spring-boot` group/artifact for Fuse 7.8. It was removed with https://issues.redhat.com/browse/ENTESB-14419

Reply:
argh, understood. In fact, we must change the template and provide a new mappign mzechanism as the [fabric8-maven-plugin](https://github.com/jboss-fuse/spring-boot-camel-xa/blob/a6811404fe8524b9720300ea1040efa462e3ec02/pom.xml#L167) is replaced by [openshift-maven-plugin](https://github.com/jboss-fuse/spring-boot-camel-xa/blob/96e6dcd5cce5692525fb9b9b4f680a4f860a1bc8/pom.xml#L171)

Reply:
wait, no, this is only for 7.9, in 7.8 it is still the fabric8-maven-plugin to usehttps://github.com/jboss-fuse/spring-boot-camel-xa/blob/a60b504cc1020cad4d93326cbf8391b16ec71068/pom.xml#L167

Reply:
I think that it should then be the 3.0.12 as it is used here https://github.com/jboss-fuse/redhat-fuse/blob/fddb37af2b38c34dbff179ca18c34d9b31922b4b/pom.xml#L43
As a reminder, this "intermediate" maven plugin is not used by default. it is kept only for compatibility for external  users having their own template.

Reply:
@apupier but mind that `version.fabric8` property is used ONLY in this fragment:
```xml
<!-- TODO: ideally this should be moved to Arquillian -->
<dependency>
	<groupId>io.fabric8</groupId>
	<artifactId>fabric8-utils</artifactId>
	<version>${version.fabric8}</version>
</dependency>
<dependency>
	<groupId>io.fabric8</groupId>
	<artifactId>kubernetes-assertions</artifactId>
	<version>${version.fabric8}</version>
	<scope>test</scope>
</dependency>
                                                          
<dependency>
	<groupId>io.fabric8</groupId>
	<artifactId>fabric8-arquillian</artifactId>
	<version>${version.fabric8}</version>
	<scope>test</scope>
</dependency>
```

and `mapping.put(FUSE_780_CAMEL_VERSION, "3.0.11.fuse-770009-redhat-00002");` is about BOM version. So I don't think we should use `3.0.12'.

Reply:
I do not understand the link with the `version.fabric8` property used in the mentioned fragment. Also from where the fragment is coming from?

I see 2 solutions:
- we use the community version 3.0.12
- we stop supporting potential templates provided by customer that are using `io.fabric8:fabric8-project-bom-camel-spring-boot` (in this case, we do not add a line for Fuse 8)

Using the 7.7 bom version for the Fuse 7.8 is leading to problem and must be avoided.

Reply:
PR for second solution https://github.com/lhein/jbosstools-fuse/pull/9